db/seeds.rb

```diff
@@ -1,7 +1,28 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+
```

Comment:
To prevent duplicate records in your deployment, you can start your seeds file with `destroy_all` commands:

Pet.destroy_all
AdoptionApplication.destroy_all
etc...

That way, your seeds file won't create multiple. You can't run `db:drop` on Heroku because it's a protected command so this is an easier option!